src/crimson/os/seastore/onode_manager/staged-fltree/node_extent_accessor.h

```diff
@@ -580,14 +568,11 @@ class NodeExtentAccessorT {
     return c.nm.retire_extent(c.t, std::move(extent)
     ).handle_error_interruptible(
       eagain_iertr::pass_further{},
-      crimson::ct_error::input_output_error::assert_failure(
-          [FNAME, c, addr] {
-        SUBERRORT(seastore_onode, "EIO -- addr={}", c.t, addr);
-      }),
-      crimson::ct_error::enoent::assert_failure(
-          [FNAME, c, addr] {
-        SUBERRORT(seastore_onode, "ENOENT -- addr={}", c.t, addr);
-      })
+      crimson::ct_error::input_output_error::assert_failure(fmt::format(
+        "{} EIO -- addr={}", FNAME, addr).c_str()),
+      crimson::ct_error::enoent::assert_failure(fmt::format(
+        "{} EIO -- addr={}", FNAME, addr).c_str()
```

Comment:
nit, EIO -> ENOENT.

If the error handler can print the error type, the more general `assert_all()` can be used.